src/api/v1/community_businesses/schema.ts

```diff
@@ -41,16 +41,28 @@ export const visitActivitiesPutPayload = {
 };
 
 export const cbPayload = {
-  name: Joi.string().min(1),
-  region: Joi.string().min(1),
-  sector: Joi.string().min(1),
-  logoUrl: Joi.string().uri(),
-  address1: Joi.string().min(1),
-  address2: Joi.string().min(1),
-  townCity: Joi.string().min(1),
-  postCode: Joi.string().min(6).max(10),
-  turnoverBand: Joi.string().min(5).max(11),
-  _360GivingId: Joi.string().min(5),
+  name: Joi.string().min(1).trim(),
+  region: Joi.string().min(1).trim(),
+  sector: Joi.string().min(1).trim(),
+  logoUrl: Joi.string().uri().trim(),
+  address1: Joi.string().min(1).trim(),
+  address2: Joi.string().min(1).trim(),
+  townCity: Joi.string().min(1).trim(),
+  postCode: Joi.string().min(6).max(10).trim(),
+  turnoverBand: Joi.string().min(5).max(11).trim(),
+  _360GivingId: Joi.string().min(5).trim(),
 };
 
 export const meOrId = id.allow('me');
+
+export const volunteerActivty = Joi.string().trim();
+
+export const volunteerDuration = Joi.object({
+  hours: Joi.number().integer().min(0),
+  minutes: Joi.number().integer().min(0),
+  seconds: Joi.number().integer().min(0),
+});
+
+export const volunteerProject = Joi.string().min(2).max(255);
+
+export const volunteerProjectOrNull = Joi.alt().try(volunteerProject, Joi.only(null));
```

Comment:
Don't think this is necessary? Shouldn't `Joi.string().min(2).max(255).allow(null)` work too? Then you can just specify nullability directly in the route definition, e.g.

```
...
validate: {
  payload: {
    ...
    project: volunteerProject.allow(null),
    ...
  }
}
...
```